packages/ibm-products/src/components/CoachmarkStack/CoachmarkStackHome.tsx

```diff
@@ -96,183 +95,170 @@ const componentName = 'CoachmarkStackHome';
 export let CoachmarkStackHome = forwardRef<
   HTMLDivElement,
   CoachmarkStackHomeProps
->(
-  (
-    {
-      className,
-      description,
-      isOpen,
-      renderMedia,
-      navLinkLabels,
-      onClickNavItem,
-      onClose,
-      portalTarget,
-      closeButtonLabel,
-      title,
-      tooltipAlign,
-      ...rest
-    },
-    ref
-  ) => {
-    const buttonFocusRef = useRef<ButtonProps<React.ElementType> | null>(null);
-    const [linkFocusIndex, setLinkFocusIndex] = useState(0);
-    const navItemRefs = useRef<(HTMLLIElement | null)[]>([]);
-    const [overflowStates, setOverflowStates] = useState<boolean[]>([]);
+>((props, ref) => {
+  const {
+    className,
+    description,
+    isOpen,
+    renderMedia,
+    navLinkLabels,
+    onClickNavItem,
+    onClose,
+    portalTarget,
+    closeButtonLabel,
+    title,
+    tooltipAlign,
+    ...rest
+  } = props;
+  const buttonFocusRef = useRef<ButtonProps<React.ElementType> | null>(null);
+  const [linkFocusIndex, setLinkFocusIndex] = useState(0);
+  const navItemRefs = useRef<(HTMLLIElement | null)[]>([]);
+  const [overflowStates, setOverflowStates] = useState<boolean[]>([]);
 
-    useEffect(() => {
-      setTimeout(() => {
-        if (isOpen && buttonFocusRef.current) {
-          buttonFocusRef.current.focus();
-        }
-      }, 100);
-    }, [isOpen]);
-
-    const portalNode = useRef<Element | null>(null);
+  useEffect(() => {
+    setTimeout(() => {
+      if (isOpen && buttonFocusRef.current) {
+        buttonFocusRef.current.focus();
+      }
+    }, 100);
+  }, [isOpen]);
 
-    useIsomorphicEffect(() => {
-      portalNode.current = portalTarget
-        ? (document?.querySelector(portalTarget) ??
-          document?.querySelector('body'))
-        : document?.querySelector('body');
-    }, [portalTarget]);
+  const portalNode = useRef<Element>(null);
 
-    if (!navLinkLabels) {
-      return pconsole.warn(
-        `${componentName} is an Onboarding internal component and is not intended for general use.`
-      );
+  useEffect(() => {
```

Comment:
Will this cause SSR issues? It was a while back, but didn't we move to using `useIsomorphicEffect` because of that?

Reply:
oh interesting. let me put that back